### DISCUSSION TITLE
can't select all nodes on live example

### DISCUSSION THREAD

**Discussion by rimzzlabs** [mdx-editor/editor#21] [Q&A] [answered]
Hello! when I was wandering around and trying out the live example, I could not select all nodes with `ctrl + a`, because I wanted to clear the content and start from nothing, but It didn't clear the content or nodes, just freezing, and the caret was also going back behind the first node on `| Welcome`. like that

If you visit [the live example](https://mdxeditor.dev/editor/demo) and select all content, you just can't select them all with `ctrl + a`.

**Comment by petyosi** [comment]
Confirming, that's a bug in Lexical, I am [watching this issue](https://github.com/facebook/lexical/issues/4358). If it's not resolved anytime soon, I will apply the workaround suggested.

**Comment by petyosi** [comment, marked as answer]
Fixed in v0.15.0.
